neo/test/rawiotest/test_neuralynxrawio.py

```diff
@@ -82,9 +154,141 @@ def test_recording_types(self):
         for typeTest in self.ncsTypeTestFiles:
 
             filename = self.get_filename_path(typeTest[0])
-            hdr = NlxHeader.build_for_file(filename)
+            hdr = NlxHeader(filename)
             self.assertEqual(hdr.type_of_recording(), typeTest[1])
 
 
+class TestNcsSectionsFactory(TestNeuralynxRawIO, unittest.TestCase):
```

Comment:
I think that each class that inherits TestNeuralynxRawIO will run the BaseTestRawIO on all files again and again.

You should :
  * add `entities_to_test = []` here to avoid this.
  * or not create a new class but put every test in one class only.

Reply:
Ah good point. I just pushed a change for that. Let's see if that passes on Circle now.

Reply:
Looks like that solved the timeout problem, thanks. All other conversations closed and fine by me to merge now.